src/test/tools/extents/README

```diff
@@ -5,7 +5,11 @@ This is src/test/tools/extents/README.
 This directory contains a tool for listing physical extents of a file.
 
 Usage:
-	$ extents file
+	$ extents file [logical-offset]
 
-Output of this command is a simple list of pairs: offset and length expressed
-in sectors (offset from the beginning of the device).
+If no 'logical-offset' is given, than the output of this command
+is a simple list of pairs: physical offset and length - both of them
+expressed in 512B sectors (offset from the beginning of the device).
+
+If a 'logical-offset' is given, than the output of this command
+is the physical offset of the given logical one.
```

Comment:
I think it makes more sense to implement this option as a filter that narrows the list of extents to the chosen offset. The problem of useless "length" could be resolved by adding another argument that disables it.

Reply:
Done